src/screens/guides/components/ecology-recipe.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Ecology from "ecology";
+import { ecologyPlaygroundLoading } from "formidable-landers";
+
+class EcologyRecipe extends React.Component {
+  customRenderers(pathname) {
+    return {
+      heading: (text, level) => {
+        const escaped = text.toLowerCase().replace(/[^\w]+/g, "-");
+
+        return `<h${level} id="${escaped}"><a class="Anchor" href="${pathname}#${escaped}" aria-hidden="true"></a>${text}</h${level}/>`;
+      },
+      ...ecologyPlaygroundLoading
+    };
+  }
+
+  render() {
+    const { scope, overview, location } = this.props;
+    const pathname = location.pathname;
+
+    return (
+      <div className="Recipe Markdown">
```

Comment:
Note that this `div` is now rendered with the additional `Markdown` class, which grants the styles necessary for rendering a clickable anchor. Conferred with @paulathevalley about whether this might introduce any issues and we think not. Didn't notice any when testing locally.